webpack/scenes/ContentViews/Details/Filters/ContentViewFilters.js

```diff
@@ -32,6 +33,9 @@ const ContentViewFilters = ({ cvId }) => {
   const [searchQuery, updateSearchQuery] = useState('');
   const loading = status === STATUS.PENDING;
   const [addModalOpen, setAddModalOpen] = useState(false);
+  const [bulkActionOpen, setBulkActionOpen] = useState(false);
+  const [bulkActionEnabled, setBulkActionEnabled] = useState(false);
+  const toggleBulkAction = () => setBulkActionOpen(!bulkActionOpen);
```

Comment:
```suggestion
  const toggleBulkAction = () => setBulkActionOpen(prevBulkActionOpen => !prevBulkActionOpen);
```
If a state change depends on the previous state, it's always safer to use the function form of `setState` :)

Reply:
Can you explain why this would be dangerous? I'm not seeing the relevance of this.

Reply:
It may be fine in this particular case since it's just a boolean, but it's possible that the `bulkActionOpen` value may be stale if you don't use the function form. I think @sjha4 actually did run into some bugs around that once.